flottform/demo/src/routes/custom-ui/+page.svelte

```diff
@@ -0,0 +1,586 @@
+<script lang="ts">
+	import { base } from '$app/paths';
+	import { onMount } from 'svelte';
+	import { FlottformFileInputHost } from '@flottform/forms';
+	import { writable } from 'svelte/store';
+	import FileInput from '$lib/components/FileInput.svelte';
+	import { env } from '$env/dynamic/public';
+	import { browser } from '$app/environment';
+	import { sdpExchangeServerBase } from '../../api';
+	import {
+		closeDialogButtonCss,
+		closeSvg,
+		createChannelButtonCss,
+		createChannelElementCss,
+		createChannelQrCodeCss,
+		createChannelStatusWrapperCss,
+		dialogCss,
+		defaultStyles,
+		flottformSvg,
+		refreshConnectionButtonCss,
+		simulateHoverEffect,
+		type Styles,
+		changeBackgroundOnState
+	} from '../../../../forms/src/flottform-styles';
```

Comment:
Let's see if we can easily use Flottform events to change UI directly in our framework instead of using plain JS to create components and change styles 🤔 
```suggestion

```